src/seedu/addressbook/data/person/Name.java

```diff
@@ -60,5 +60,13 @@ public boolean equals(Object other) {
     public int hashCode() {
         return fullName.hashCode();
     }
+    
+    /**
+     * Returns true of the other name is very similar to this name.
+     * Two names are considered similar if ...
+     */
+     public boolean isSimilar(Name other) {
```

Comment:
First commit should be a commit showing your first test case : Function not yet implemented.